index.html

```diff
@@ -39,7 +39,124 @@ <h2 class="caption">I’am a Software Developer</h2>
                 </ul>
             </div>
         </section> 
-        <div class="side-img"></div> 
     </main>
+    <section class="works">
+        <div class="title">
+            <h2>My Recent Works</h2>
+            <div class="line-break"></div>
+        </div>
+        <section class="card-works">
+            <div class="card-work1">
+                <img src="./images/Placeholder.png" alt="Resturant Project">
+                <h3>Multi-Post Stories</h3>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                <ul>
+                    <li>css</li>
+                    <li>html</li>
+                    <li>bootstrab</li>
+                    <li>Ruby</li>
+                </ul>
+                <button type="button">See Project</button>
+            </div>
+            <div class="card-work">
+                <h3>Profesional Art<br> Printing Data</h3>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul>
+                    <li>html</li>
+                    <li>bootstrab</li>
+                    <li>Ruby</li>
+                </ul>
+                <button type="button">See Project</button>                
+            </div>
+            <div class="card-work">
+                <h3>Profesional Art<br> Printing Data</h3>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul>
+                    <li>html</li>
+                    <li>bootstrab</li>
+                    <li>Ruby</li>
+                </ul>
+                <button type="button">See Project</button>                
+            </div>
+            <div class="card-work">
+                <h3>Profesional Art<br> Printing Data</h3>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul>
+                    <li>html</li>
+                    <li>bootstrab</li>
+                    <li>Ruby</li>
+                </ul>
+                <button type="button">See Project</button>                
+            </div>
+            <div class="card-work">
```

Comment:
- [ ] For each project, add a margin on all sides so that your design matches the given template design.